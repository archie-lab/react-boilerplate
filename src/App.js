import React, {Component} from "react";
import {hot} from "react-hot-loader/root";
import "./App.scss";

class App extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return <p className="app">Hello</p>;
  }
}

export default hot(App);
