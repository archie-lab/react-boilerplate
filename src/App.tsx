import * as React from "react";
import { hot } from "react-hot-loader/root";
import "./App.scss";

class App extends React.Component<{}> {
  render() {
    return <p className="app">Hello</p>;
  }
}

export default hot(App);
