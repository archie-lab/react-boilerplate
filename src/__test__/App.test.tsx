import * as React from "react";
import {shallow} from "enzyme";
// import Adapter from "enzyme-adapter-react-16";

import App from "../App";

// configure({adapter: new Adapter()});

describe("App Component", () => {
  it("should exist", () => {
    expect(shallow(<App />)).toBeTruthy();
  });

  it("should exist", () => {
    expect(shallow(<App />)).toBeTruthy();
  });

  it("should have one heading", () => {
    expect(
      shallow(<App />)
        .find(".app")
        .type()
    ).toEqual("p");
  });

  it("should match snapshot", () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });
});
