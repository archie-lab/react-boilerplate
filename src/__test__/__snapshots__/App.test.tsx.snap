// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should match snapshot 1`] = `
<p
  className="app"
>
  Hello
</p>
`;
